basic/oneliner

```diff
@@ -13,6 +14,24 @@
 
 ../c99sh -mse 'printf("Hello from 1-liner\n");' </dev/null
 
+# Using -F causes the option to appear
+echo 'printf(MACRO);' | ../c99sh -ms '-FDMACRO="Salut\n"'
+
+# Further, using -F appends to CFLAGS
+(
+    export 'CFLAGS=-DANOTHER="Bonjour"'
+    echo 'printf(ANOTHER MACRO);' | ../c99sh -ms '-FDMACRO="Salut\n"'
+)
+
+# Using -L causes the option to appear thereby *breaking* this example
+! ../c99sh -mse 'printf("First test of -L\n");' -Llnonexistent </dev/null
```

Comment:
Easier to come up with ways to break with linking lines then to quickly confirm behavioral changes via linking.